### ISSUE TITLE
How do you handle post-processing of your photos?

### ISSUE THREAD

**Issue by pmbanka** [Krzysztof-Cieslak/AMA#17]
I have an an entry-level DSLR and I like shooting some pictres, mostly during holidays (and at the same time I shoot some pictures with my phone as well). When I'm back from holiday, I end up with a pile of photos which need to be cleaned and organized... somehow. This task is so overwhelming that I have a backlog of photos from at least last 2 years...

So my question is - how do you tackle this problem? How do you organize photos on hdd? What tools do you use? How do you backup? How do you choose which photos you should delete and which ones you should process further? Where do you keep photos after post processing, for convenient access and sharing? Do you store all the photos that you take? How much time does it all take?

tl;dr please share what is your process between shooting a picture and having it displayed somewhere :)

**Comment by Krzysztof-Cieslak** [comment]
> How do you organize photos on hdd?
I'm putting photos in the folders based on the "trip", putting date and place in the folder name (YYYYMMDD-MMDD_NameOfPlace) which gives me right sorting in the default alphabetic sort in Windows,

>What tools do you use?
For post processing I'm using Adobe Lightroom - It's really powerful, UX is decent enough. I'm pretty conservative and against typical photo manipulation so I don't need capabilities of Photoshop

> How do you backup?
I just have few copies of my folder with all photos on several external hard drives, nothing fancy.

> How do you choose which photos you should delete and which ones you should process further? Do you store all the photos that you take?
I keep most of my photos, I delete only ones that are clearly terrible (also, I do quite a lot of initial assessment on the camera, during the trip - recently I've been trying hard to focus on having less, but more interesting photos instead of having tones of photos no one watches later anyway). I do quick post processing on most of the photos I keep ( fixing some clear problems - white balance, exposure). I spend more time on the photos I want to print, publish or show to someone.

> Where do you keep photos after post processing, for convenient access and sharing? 
I shoot in and process RAWs so the post processing is not really changing the file - result of the post processing is small `xml` containing all changes which is automatically put next to original file (which means I backup it normally). I only burn out JPGs when I want to upload stuff to 500px, or print them (BTW, you should print photos!).

> How much time does it all take?

It really depends. Rough, quick post processing is bit of automatic process (and often no changes are really needed). For the pictures I want to show it varies a lot, I sometimes spend hours on one photo trying out different post-processing ideas I had for the photo, sometimes I have clear idea what I want and it's pretty fast.
